bitnami/apache/templates/prometheusrules.yaml

```
@@ -0,0 +1,27 @@
+{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled .Values.metrics.prometheusRule.rules }}
```

Shouldn't we simply check the 1st two conditions? I'd simply throw an error and abort the installation if the 1st two conditions are met and `.Values.metrics.prometheusRule.rules` is empt.y

---

yes, i have simplify to the 1st two condition.